### PR TITLE
[Components with more than 1 service] LPS-176900, LPS-177016, LPS-177034

### PR DESCRIPTION
**Pull request by daiazius**
Related issues: https://issues.liferay.com/browse/LPS-176900, https://issues.liferay.com/browse/LPS-177016, https://issues.liferay.com/browse/LPS-177034

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
@rafaprax

**Comment by catarineaguiar** [comment]
no regressions found.

PS: this [unique failure](https://testray.liferay.com/home/-/testray/case_results/2230629542?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcases%2F1858779038%2Fview%3Fcur%3D1%26delta%3D200%26testrayCaseId%3D1858779038%26orderByCol%3DcreateDate_sortable%26orderByType%3Ddesc) doesn't seem related

**Comment by rafaprax** [comment]
@daiazius  Let's apply the same pattern here https://github.com/liferay-core-infra/liferay-portal/pull/3866

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
Pushed with no changes in the files, I've only changed the commit pattern

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/dto/v1_0/converter/AccountMemberDTOConverter.java [hunk 1]
```diff
@@ -91,7 +91,9 @@ private AccountRole[] _getAccountRoles(
 		return stream.toArray(AccountRole[]::new);
 	}
 
-	@Reference
+	@Reference(
+		target = "(component.name=target = com.liferay.headless.commerce.admin.account.internal.dto.v1_0.converter.AccountRoleDTOConverter)"
+	)
```
Comment (rafaprax) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/dto/v1_0/converter/AccountMemberDTOConverter.java:96]: Hey @daiazius, you forgot to change to DTOConverter here. please fix it.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey @tinatian, this pr is ready for your review. Thx

**Comment by tinatian** [comment]
@rafaprax @daiazius  Does this need to retrigger the test? I did not see any comment saying what's changed in the force push.

Tina

**Comment by tinatian** [comment]
@daiazius "I just corrected a reference that I forgot to change", this kind of change needs to retrigger the tests, as you actually changed your code.

The only case we did not retrigger is code conflicts caused by baseline related stuff or generated stuff, as long as you actually modified your code, please retrigger the tests.

@rafaprax Please make sure tests are retriggered for any code change.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
Portal startup WARN:
```
2023-03-17 20:39:32.327 WARN  [main][SystemCheckOSGiCommands:154] Declarative Service Unsatisfied Component Checker check result: 
Bundle {id: 404, name: com.liferay.headless.commerce.admin.account.impl, version: 4.0.63}
	Declarative Service {id: 1590, name: com.liferay.headless.commerce.admin.account.internal.resource.v1_0.AccountMemberResourceImpl, unsatisfied references: 
		{name: _accountMemberDTOConverter, target: (component.name=com.liferay.headless.commerce.admin.account.internal.dto.v1_0.converter.AccountMemberDTOConverter)}
	}
	Declarative Service {id: 1599, name: com.liferay.headless.commerce.admin.account.internal.resource.v1_0.factory.AccountMemberResourceFactoryImpl, unsatisfied references: 
		{name: _componentServiceObjects, target: null}
	}
	Declarative Service {id: 1558, name: com.liferay.headless.commerce.admin.account.internal.dto.v1_0.converter.AccountMemberDTOConverter, unsatisfied references: 
		{name: _accountRoleDTOConverter, target: (component.name=target = com.liferay.headless.commerce.admin.account.internal.dto.v1_0.converter.AccountRoleDTOConverter)}
	}
	Declarative Service {id: 1562, name: com.liferay.headless.commerce.admin.account.internal.graphql.servlet.v1_0.ServletDataImpl, unsatisfied references: 
		{name: _accountMemberResourceComponentServiceObjects, target: null}
```
Full console:
https://testray.liferay.com/reports/production/logs/2023-03/test-1-27/test-portal-acceptance-pullrequest(master)/7354/functional-smoke-tomcat90-mysql57-jdk8_stable/0/0/jenkins-console.txt.gz?authuser=0

**Comment by daiazius** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/132067

**Pull request closed without merging**